rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ Required ] I think it would be better if you provide `Getters` and `Setters` for the `@book` and `@person` instance variables. Currently, your implementation only provides `Getters`. Please see the example below for an example of how you can achieve this.

## Example code
```ruby
  attr_accessor :date, :book, :person

```